### PR TITLE
feat: APPS-2395 og identifier plus external item record electronic locator

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-2395](https://uclalibrary.atlassian.net/jira/software/c/projects/APPS/boards/12?assignee=712020%3Af4539cc0-90d3-4445-9de9-94d4149de1e2&selectedIssue=APPS-2395)

Work
Screenshot 2024-01-05 at 9 53 12 PM

Collection:
Screenshot 2024-01-05 at 9 49 55 PM

[APPS-2395]: https://uclalibrary.atlassian.net/browse/APPS-2395?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/locales/blacklight.en.yml [hunk 1]
```diff
@@ -13,9 +13,11 @@ en:
           normalized_date_sim: 'Normalized Date'
           rights_country: 'Rights (country of creation)'
           services_contact: 'Rights services contact'
+          electronic_locator: 'External item record'
```
Comment (pghorpade) [inline config/locales/blacklight.en.yml:16]: we dont need this on the index page.

File: config/locales/hyrax.en.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ en:
           publisher_tesim: Publisher
           rights_statement_tesim: Rights Statement
           subject_tesim: Subject
+          electronic_locator: External item record
```
Comment (pghorpade) [inline config/locales/hyrax.en.yml:32]: remove from index page.

File: config/locales/hyrax.en.yml [hunk 3]
```diff
@@ -64,6 +66,7 @@ en:
         rights_holder: 'Rights Holder'
         rights_statement: 'Copyright Status'
         services_contact: 'Rights services contact'
+        electronic_locator: External item record
```
Comment (pghorpade) [inline config/locales/hyrax.en.yml:69]: remove it from under simple form.

File: config/locales/simple_form.en.yml [hunk 4]
```diff
@@ -9,3 +9,4 @@ simple_form:
       rights_country: 'Rights (country of creation)'
       rights_holder: 'Rights Holder'
       services_contact: 'Rights services contact'
+      electronic_locator: 'External item record'
```
Comment (pghorpade) [inline config/locales/simple_form.en.yml:12]: why are we adding it here?

File: spec/system/import_and_show_work_spec.rb [hunk 5]
```diff
@@ -56,6 +56,7 @@
     expect(page).to have_content "uclamss_1387_b112_40911-1" # local_identifier
     expect(page).to have_content "[between 1942-1947]" # date_created
     expect(page).to have_content "Editor-1"
+    expect(page).to have_content "http://test.url/external/item/record" # electronic_locator
```
Comment (pghorpade) [inline spec/system/import_and_show_work_spec.rb:59]: should it be testing for identifier content too?

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: config/locales/simple_form.en.yml (hunk 4 above)
Comment (jendiamond) [reply config/locales/simple_form.en.yml:12]: I thought this was the edit form.

File: spec/system/import_and_show_work_spec.rb (hunk 5 above)
Comment (jendiamond) [reply spec/system/import_and_show_work_spec.rb:59]: I added it.

**Review by pghorpade — APPROVED** [review]
👍

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**